link/src/main/java/com/stripe/android/link/ui/signup/SignUpScreen.kt

```diff
@@ -195,11 +199,14 @@ private fun PhoneCollectionSection() {
             style = MaterialTheme.typography.caption
         )
         Button(
-            onClick = {},
+            onClick = {
+                onSignUpClick(phone)
+            },
             modifier = Modifier
                 .fillMaxWidth()
                 .height(56.dp)
-                .padding(vertical = 8.dp)
+                .padding(vertical = 8.dp),
+            enabled = phone.length == 10
```

Comment:
I am curious why is this only enabled if the phone length is 10, I thought maybe other countries had different phone lengths?